cargo-workspaces/src/utils/version.rs

```diff
@@ -219,6 +248,62 @@ impl VersionOpt {
         Ok(())
     }
 
+    fn alert_unversioned(
+        &self,
+        pkgs: HashMap<&str, (&str, Vec<(&str, &VersionReq, &Version)>)>,
+    ) -> Result {
+        if pkgs.is_empty() {
+            return Ok(());
+        }
+        loop {
+            match Select::with_theme(&ColorfulTheme::default())
+                .with_prompt(format!(
+                    "You have {} packages with unversioned dependencies",
+                    pkgs.len()
+                ))
+                .items(&["Review Dependencies", "Auto-version", "Abort"])
+                .default(0)
+                .clear(true)
+                .interact_on_opt(&TERM_ERR)?
+            {
+                Some(2) | None => exit(0),
+                Some(1) => {
+                    if Confirm::with_theme(&ColorfulTheme::default())
+                        .with_prompt(
+                            "Are you sure you want this tool to auto-inject these versions?",
+                        )
+                        .default(false)
+                        .interact_on(&TERM_ERR)?
+                    {
+                        return Ok(());
+                    }
+                }
+                _ => {
+                    let mut items = vec![];
+                    for (name, deps) in pkgs.values() {
+                        items.push(format!(" │ {}", style(name).green()));
+                        for (dep, ver, new_ver) in deps {
+                            items.push(format!(
+                                " │ \u{21b3} {}: {} => {}",
+                                style(dep).cyan(),
+                                style(ver).yellow(),
+                                style(new_ver).green()
+                            ));
+                        }
+                    }
+                    Select::new()
```

Comment:
Do you think there's a better way to show this? Using `less` or `pager` or something? Using the `select` prompt can seem a bit confusing to the user.

Reply:
Hm, I actually thought it was a clever use of the select prompt, dialoguer's pager thankfully also shows a page count `[Page 1/33]`, the left `>` indicator serves as a helpful scrollbar. But you're the maintainer, if you prefer an actual pager that's alright. Although, how then do you ensure behavioral consistency across platforms? I'm not versed in the pagers ecosystem for windows. Any ideas?

Reply:
\

> Don't think so. This is only supposed to happen once, right?

I think that this would happen frequently.  In my workspace, I have the utilities pointed to by path in the bins' Cargo.toml's.  what i would like is any time a utility that is pointed to without a version (eg `util_config = { path = "../util_config" }`), it will always point to the latest commit with a change in that utility